app/assets/javascripts/initializers/initializeFetchFollowedArticles.js.erb

```diff
@@ -23,64 +23,12 @@ function insertArticle(article,parent,insertPlace) {
 
 function insertInitialArticles(user) {
   var el = document.getElementById("home-articles-object");
-  if (!el) {return}
+ if (!el) {return}
   el.innerHTML = "";
-  insertNewArticles(user);
   insertTopArticles(user);
   el.outerHTML = "";
 }
 
-function insertNewArticles(user){
```

Comment:
Are new articles being removed, because the new feed endpoint will just put it all in top articles?

Reply:
Exactly!